.github/workflows/ci.yml

```diff
@@ -30,12 +30,12 @@ jobs:
       - uses: actions/setup-node@v3
         with:
           node-version-file: '.nvmrc'
-          cache: yarn
+          cache: npm
 
       - name: Build Storybook docs
         run: |
-          yarn install
-          yarn build-storybook --quiet
+          npm install
+          npm run storybook --quiet
```

Comment:
ah, this is why it hangs :-) this is starting the SB dev server locally, but it should make a static build of storybook instead!